src/generic_ui/polymer/roster.html

```diff
@@ -63,7 +69,7 @@
     </div>
 
     <div hidden?='{{ !model.onlineNetwork.hasContacts }}'>
-      <input type='text' id='search' placeholder='Search' value='{{ searchQuery }}'></input>
+      <input type='text' id='search' placeholder='Search' value='{{ searchQuery }}' hidden></input>
```

Comment:
Can you add a comment that search is disabled for now intentionally but we are leaving the code in so it can be readded if we change our mind?